test/docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG GO_IMAGE=golang:1.13-stretch
+ARG GO_IMAGE=golang:1.14
```

Comment:
Why change this?

Reply:
I got an error saying that 1.14 is the minimum required golang version to build algod. I believe this requirement is enforced here, https://github.com/algorand/go-algorand/blob/master/scripts/get_golang_version.sh

Reply:
Ok, that makes sense